shared/css/sidebar.css

```diff
@@ -0,0 +1,93 @@
+.media-wrapper:not(.showing-local-streams) > .media-control-buttons {
```

Comment:
Licenses missing. Plus I feel like we're going to need to add a post-landing review/tidy up of css